ppx_import.opam

```diff
@@ -12,7 +12,7 @@ dev-repo: "git+https://github.com/ocaml-ppx/ppx_import.git"
 tags: [ "syntax" ]
 
 depends: [
-  "ocaml"                   {              >= "4.04.2" & < "4.10.0" }
+  "ocaml"                   {              >= "4.04.2" & <= "4.10.0" }
```

Comment:
This should be `< "4.11.0"` to work with future bugfix releases of 4.10.

Reply:
Is the API guaranteed to be stable in minor releases?

Reply:
[Note that the right constrain is usually `< "4.11"` as to cover beta and dev releases.

Reply:
There is no such guarantee that I know of, but in practice it is usually stable; in fact I don't know of a single .1 release that changed the syntax API.  And you don't want to have to do more work on new minor releases by default.